hw/ip/otbn/util/yaml_to_doc.py

```diff
@@ -134,6 +135,18 @@ def render_encoding(mnemonic: str, encoding: Encoding) -> str:
     return ''.join(parts)
 
 
+def render_literal_pseudo_op(rewrite: List[str]) -> str:
+    '''Generate documentation with expansion of a pseudo op'''
+    parts = []
+    parts.append('This instruction is a pseudo-operation and expands to the '
+                 'following RV32I instruction sequence:\n```\n')
```

Comment:
Good point: this shouldn't appear in the generated docs. It *is* important that we generate RV32I instructions here, because we pass the results straight to binutils rather than assembling them a second time. But I agree that a reader of the ISA shouldn't care!